packages/slate-theme/src/scripts/sections/product.js

```diff
@@ -29,47 +29,42 @@ theme.Product = (function() {
    */
   function Product(container) {
     this.$container = $(container);
-    var sectionId = this.$container.attr('data-section-id');
-
-    this.settings = {};
-    this.namespace = '.product';
 
     // Stop parsing if we don't have the product json script tag when loading
     // section in the Theme Editor
     if (!$(selectors.productJson, this.$container).html()) {
       return;
     }
 
+    var sectionId = this.$container.attr('data-section-id');
+    var $featuredImage = $(selectors.productFeaturedImage, this.$container);
```

Comment:
Maybe we can assign this to `this.$featuredImage` considering we're accessing this element both in the constructor (if it exists) and the `updateProductImage` method below.